README.md

```diff
@@ -1,2 +1,86 @@
-# covid-19-Tracker
-This app is used to get information about COVID-19 cases around the world.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Covid-19 Tracker
+
+
+> This is a web application built with react and redux. It uses an API to get covid statitics in African countries and renders it on the page.
+
+## Built With
+- HTML
+- Boostrap
+- JavaScript
+- React JS
+- Redux
+- React Testing library
+- Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
+- Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+- ESlint (A mighty, modern linter that helps you avoid errors and enforce conventions in JavaScript codes)
+
+## Project Screenshots
+
+![Mission page](./assets/country-desktop.png)
+![Rocket page](./assets/country-list-desktop.png)
+![Profile page](./assets/country-list-mobile.png)
+![Profile page](./assets/country-mobile.png)
+
+View project Video [here](https://www.loom.com/share/6dde5bc7703a44aba7b853fb496bdd29)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- copy [this link](https://github.com/Ibizugbe/covid-19-Tracker).
+- get the directory that you want to clone the repository.
+- open the command prompt in this directory.
+- run `git clone https://github.com/Ibizugbe/covid-19-Tracker`
+- use npm install to install the dependencies.
+- use npm start to start live server.
+
+## Prerequisites
+
+- Have a working and updated browser
+- Have a local version control like git install on your computer
+- Have an account and sign in on GitHub, as online/remote version control system
+- Basic knowledge in JS and React JS
+
+### Run tests
+
+- `npx stylelint "/*.{css,scss}"**`: To test css code
+- `npx eslint .` : To test Eslint Report
+
+### To test the application
+
+- `npm install --save-dev jest`
+- `npm test`
+
+## Live Demo
+
+- [Visit here](https://space-travellers-97.netlify.app/)
```

Comment:
- [ ] Kindly make sure you deploy the project and add the correct deployment link in your ReadMe. Currently, the link seems to take me to the space travelers project. You can use Netlify or Heroku for deployment but I personally recommend Netlify since Heroku will abandon it's free services in september.